README.md

```
@@ -141,42 +141,22 @@ PREFECT_PROXY_API_URL=
 
-#### For DBT 1.10.19
-```bash
-cd dbt_deps/dbt-1.10.19/
-UV_PROJECT_ENVIRONMENT=$DBT_VENV/venv-1.10.19 uv sync
-```
-
-**Note**: The `uv.lock` files are committed to ensure reproducible dependency versions across all environments.
-
-Set the DBT environment path in `.env`:
+Set the DBT paths in `.env`:
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)